templates/rustdoc/topbar.html

```diff
@@ -64,56 +59,56 @@
                             <li class="pure-menu-heading">Links</li>
 
                             {# If the crate has a homepage, show a link to it #}
-                            {%- if krate.homepage_url -%}
+                            {%- if let Some(homepage_url) = krate.homepage_url -%}
                                 <li class="pure-menu-item">
-                                    <a href="{{ krate.homepage_url }}" class="pure-menu-link">
-                                        {{ "house" | fas }} Homepage
+                                    <a href="{{ homepage_url }}" class="pure-menu-link">
+                                        {{ "house"|fas(false, false, "")|safe }} Homepage
                                     </a>
                                 </li>
                             {%- endif -%}
 
                             {# If the crate has external docs, show a link #}
-                            {%- if krate.documentation_url -%}
+                            {%- if let Some(documentation_url) = krate.documentation_url -%}
                                 <li class="pure-menu-item">
-                                    <a href="{{ krate.documentation_url }}" title="Canonical documentation" class="pure-menu-link">
-                                        {{ "file-lines" | far }} Documentation
+                                    <a href="{{ documentation_url }}" title="Canonical documentation" class="pure-menu-link">
+                                        {{ "file-lines"|far(false, false, "")|safe }} Documentation
                                     </a>
                                 </li>
                             {%- endif -%}
 
                             {# If a the crate has a repo url, show it #}
-                            {%- if krate.repository_url -%}
+                            {%- if let Some(repository_url) = krate.repository_url -%}
                                 <li class="pure-menu-item">
-                                    <a href="{{ krate.repository_url }}" class="pure-menu-link">
-                                        {{ "code-branch" | fas }} Repository
+                                    <a href="{{ repository_url }}" class="pure-menu-link">
+                                        {{ "code-branch"|fas(false, false, "")|safe }} Repository
                                     </a>
                                 </li>
                             {%- endif -%}
 
                             <li class="pure-menu-item">
                                 <a href="https://crates.io/crates/{{ krate.name }}" class="pure-menu-link" title="See {{ krate.name }} in crates.io">
-                                    {{ "cube" | fas }} crates.io
+                                    {{ "cube"|fas(false, false, "")|safe }} Crates.io
```

Comment:
@GuillaumeGomez was there a reason for reverting this back to capitalized? (see https://github.com/rust-lang/docs.rs/pull/2508)

Reply:
No, mistake on my end. Gonna send a PR to fix it.